src/components/pages/CreatePage.jsx

```diff
@@ -203,7 +203,16 @@ const renderObjects = (recordTypes = [], config) => {
 const renderProcedures = (recordTypes = [], config, getTagsForRecord, tagConfig) => {
   const serviceType = 'procedure';
 
-  const grouped = Object.groupBy(recordTypes, (recordType) => getTagsForRecord(recordType) || 'defaultGroup');
+  const grouped = {};
+  recordTypes.forEach((recordType) => {
+    const tag = getTagsForRecord(recordType) || 'defaultGroup';
+    if (grouped[tag] === undefined) {
+      grouped[tag] = [recordType];
+    } else {
+      grouped[tag].push(recordType);
+    }
+  });
+
```

Comment:
@axb21 This is the only thing I had a question about - I still haven't been able to replicate the issue Jessi reported about the Create New page not rendering, but Monica did and her screenshot had `Object.groupBy` not being found. I noticed when I looked at the function on [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/groupBy) they mention that this has been available since March 2024, so it's fairly new and might not be available if someone is out of date. I replaced it with a simple implementation but wanted to see if you had any thoughts about it.

Reply:
Eesh, sorry I didn't get around to commenting on this till now. I can't say I have anything to offer, unfortunately. I expect that if a function was added in March 2024 then a large fraction of potential users won't have it available to them. People don't upgrade their browsers as often as you'd hope.